### DISCUSSION TITLE
Sonoff 433 Bridge - Getting random data few times.

### DISCUSSION THREAD

**Discussion by vishalalex** [arendst/Tasmota#19342] [Q&A] [answered]
Last week i have got it Sonoff Bridge 2R2 V2.2 flashed with 13.0.0.4(sensors). I am noticing the when i press the same key on a RF remote, the data changes for just once in 4 press. otherwise it is just constant. this is same with all the buttons on the remote. this happens on the door sensor also. sometimes 3 of every data read will have random number.

<img width="960" alt="chrome_v5fY4SqLST" src="https://github.com/arendst/Tasmota/assets/47385822/048bede0-c538-4efa-9c7c-889262816a2f">

**Comment by sfromis** [comment, marked as answer]
Well, with that crude old technology, transmission errors are no surprise. Signal strength and distance can make a difference.

**Reply by vishalalex** [reply]
ok. didn't know that with distance code will change. but i tried keeping near and still have random codes once in 3 tries.

**Reply by vishalalex** [reply]
![image](https://github.com/arendst/Tasmota/assets/47385822/036c0b64-0b87-40bb-b9a2-9b4e86f7f935)
all these codes are generated while pressing the same button keeping it near to the sonoff bridge.

**Reply by sfromis** [reply]
Distance cannot make a difference to the code, but it can induce transmission errors causing wrong bits. If the transmitted signal is of bad quality, similar "bit rot" issues can happen. Or you may have a device deliberately transmitting varying codes for whatever reason. You might also have a noisy environment with other radio sources congesting the narrow band.

For stable Tasmota operation, It is *strongly* recommended that you have a working setup for real time, by default access to public NTP servers. You had that in first screenshot but not now.
